ggml.c

```diff
@@ -15759,7 +15759,7 @@ static void ggml_compute_backward(struct ggml_context * ctx, struct ggml_tensor
             {
                 if (src0->grad) {
                     assert(src1->type == GGML_TYPE_I32);
-                    assert(ggml_nelements(src1) == 4);
+                    assert(ggml_nelements(src1) == 3);
```

Comment:
Shouldn't this be 6? Based on the code immediately after it should be at least 4, I think, not 3.

Reply:
I went through the code an I also can't see why it's 3 when the lines just below it show it clearly taking 4 elements and looks like it designed to fail the assertion

Reply:
@ggerganov

Reply:
Should be fixed in https://github.com/ggerganov/llama.cpp/commit/513f8619535a64fa9ace808cdcbcf66211535f5c